lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -518,14 +518,7 @@ def find_bad_files_from_kubectl_output(line)
     end
 
     def confirm_context_exists
-      out, err, st = kubectl.run("config", "get-contexts", "-o", "name",
-        use_namespace: false, use_context: false, log_failure: false)
-      available_contexts = out.split("\n")
-      if !st.success?
-        raise FatalDeploymentError, err
-      elsif !available_contexts.include?(@context)
-        raise FatalDeploymentError, "Context #{@context} is not available. Valid contexts: #{available_contexts}"
-      end
+      kubectl.config_for_context(@context) # Raises if context not found
```

Comment:
The error message is now a bit less informative since it doesn't list all of the available contexts.